src/main_app/jobs_workers/public_jobs_workers/fix_nested_jobs/objects.py

```
@@ -0,0 +1,107 @@
+        "fix": {"status": "Pending", "message": "Fixing nested tags"},
+        "verify": {"status": "Pending", "message": "Verifying fixes"},
+        "upload": {"status": "Pending", "message": "Uploading fixed files"},
+    },
+}
+
+__all__ = [
+    "FixNestedJobsWorkerObject",
+    "StageDetail",
+    "Stages",
+    "FileResult",
+]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)